.github/workflows/build-wheels.yml

```diff
@@ -98,7 +98,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.8
+          python-version: "3.10"
```

Comment:
Do we have any test left that the code builds fine with Python 3.7 -> 3.9? Or should we add one?

Reply:
Sorry, I missed the top comment. Does this also mean that I can not install from a local checkout with an older Python version? If so, this might be a bit early to do this, given that most users still install from source for now, and most HPC centers do not come with Python 3.10

Reply:
Yes, Python 3.7 is not EOL until June this year, and we might want to continue supporting it for a bit longer after that

Reply:
Okay lets see what the new tests are saying. I am also fine if we keep this PR stale until we drop the versions.

Reply:
I would rather close than keep a stale PR, but we can open an issue to track this, pointing to the changes in this PR